src/components/QueryEditor/DatasetSelector.tsx

```diff
@@ -13,6 +13,7 @@ interface DatasetSelectorProps extends ResourceSelectorProps {
   applyDefault?: boolean;
   disabled?: boolean;
   onChange: (v: SelectableValue) => void;
+  inputId?: string | undefined;
```

Comment:
nit: Do we need here both `?` and `undefined`? 🤔

Reply:
good catch, removed in the latest commit